### DISCUSSION TITLE
Tag pomodoro timer

### DISCUSSION THREAD

**Discussion by masch** [focustimerhq/FocusTimer#660] [Ideas]
Hi!

I use your application for almost any kind of task that I do on my computer, and I would love to have the chance to tag the type of task that I did on a pomodoro timer.
It'll give the possibility to know with the stats view how much time I spent on each type of task. I think it'll improve the features of the application.
What do you think on tracking a tag on each pomodoro time?

Thanks!

**Comment by kamilprusko** [comment]
Hi @masch 

I'd like to include proper time tracking in Pomodoro. Currently you can set up Pomodoro with external service + https://github.com/gnome-pomodoro/gnome-pomodoro-tracking .

For the current year I'd like to focus on the core stuff (the timer behaviour, history, notifications, packaging, UI). After that, I think the task list / the daily agenda would be more used. I won't have bandwidth to do it for foreseeable future.

Suggestions on how it could look are welcomed. I'm concerned how to fit it into indicator menu. How many tags is usually used? Can they be listed or search-bar is needed?
